gdx/src/com/badlogic/gdx/math/FloatCounter.java

```diff
@@ -76,4 +79,18 @@ public void reset () {
 		value = 0f;
 		if (mean != null) mean.clear();
 	}
+
+	@Override
+	public String toString() {
+		return "FloatCounter{" +
+				"count=" + count +
+				", total=" + total +
+				", min=" + min +
+				", max=" + max +
+				", average=" + average +
+				", latest=" + latest +
+				", value=" + value +
+				", mean=" + mean +
```

Comment:
Can we remove `mean`? `value` already holds the `mean` value.

Reply:
done